src/s3ql/mount.py

```diff
@@ -600,8 +600,11 @@ async def run(self):
         assert self.fs is not None
 
         while not self.quit:
-            with trio.move_on_after(self.interval):
+            if self.interval == None:
```

Comment:
It has no practical consequence here, but please compare with `is None` rather than `== None` for clarity. We are checking if the object that self.interval points at is the None object, not if it has the same contents as the `None` object.

Thanks!